backend/src/main/resources/config/config.yml

```diff
@@ -38,8 +38,7 @@ projects:
       portalId: 102
       accessToken: OVERRIDE_IN_LOCAL_CONFIG
   - id: nuernberg.sozialpass.app
-    importUrl: https://data.entitlementcard.app/nuernberg-akzeptanzstellen.csv
-    pipelineName: SozialpassNuernberg
+    importUrl: ""
```

Comment:
We have to check if the pipeline name is still in the salt repo used just to ensure that the server Import will not fail because the pipeline is not existing anymore after this change

Reply:
please also remove the `pipelineName` for koblenz in this file

Reply:
https://git.tuerantuer.org/DF/salt/pulls/326